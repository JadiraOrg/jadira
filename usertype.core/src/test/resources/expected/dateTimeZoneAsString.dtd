<!ELEMENT dataset (
    DATETIMEZONEASSTRING*)>

<!ELEMENT DATETIMEZONEASSTRING EMPTY>
<!ATTLIST DATETIMEZONEASSTRING
    ID CDATA #REQUIRED
    DATETIMEZONE CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

