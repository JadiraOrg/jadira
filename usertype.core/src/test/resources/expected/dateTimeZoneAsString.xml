<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "dateTimeZoneAsString.dtd">
<dataset>
  <dateTimeZoneAsString ID="0" DATETIMEZONE="+04:00" NAME="test_0"/>
  <dateTimeZoneAsString ID="1" DATETIMEZONE="UTC" NAME="test_1"/>
  <dateTimeZoneAsString ID="2" NAME="test_2"/>
</dataset>